README.md

```diff
@@ -1,4 +1,23 @@
 # balance-static
 Rendered guide(s) at your fingertips
 
-How to use this:
+## Testing locally
+
+Make sure you have [hugo extended](https://gohugo.io/getting-started/installing/) installed, either drop `hugo.exe` in the root dir or add it to your PATH.
+
+```sh
+# Clone the repository
+git clone --recursive https://github.com/The-Balance-FFXIV/glam.git
```

Comment:
This looks good. One thing I noticed that I was using `git submodule update --init --recursive` I guess this also updates that `hermit` theme...

Reply:
prolly will have to remove that `hermit` theme sometime later.